### PR TITLE
Reinitiate Official support for ruby

### PR DESCRIPTION
**Pull request by postcumer**

### CODE REVIEW

File: rubyx.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "postcumer",
+			"oem": "Redmi",
+			"device": "Redmi Note 12 Pro 5G",
+			"filename": "crDroidAndroid-14.0-20240915-rubyx-v10.8.zip",
```
Comment (gwolf2u) [inline rubyx.json:7]: this supposed to be cr11

File: rubyx.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			"download": "https://sourceforge.net/projects/crdroid/files/rubyx/10.x/crDroidAndroid-14.0-20240915-rubyx-v10.8.zip/download",
+			"timestamp": 1726388699,
+			"md5": "78fd0e8993f248dadc2de1ec401c6a95",
+			"sha256": "02c487935c59f495f7e4c26e845b1347c3c24ef390f9e7a56f103ce7da335b1d",
+			"size": 1318758259,
+			"version": "11",
```
Comment (gwolf2u) [inline rubyx.json:13]: leave as generated

File: rubyx.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+			"buildtype": "Monthly",
+			"forum": "https://xdaforums.com/t/temp-closed-rom-15-unofficial-crdroid-v11-beta-aosp-stable.4706860/",
```
Comment (gwolf2u) [inline rubyx.json:15]: fix mod edit

File: rubyx.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+			"oem": "Redmi",
+			"device": "Redmi Note 12 Pro 5G",
+			"filename": "crDroidAndroid-14.0-20240915-rubyx-v10.8.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/rubyx/10.x/crDroidAndroid-14.0-20240915-rubyx-v10.8.zip/download",
+			"timestamp": 1726388699,
+			"md5": "78fd0e8993f248dadc2de1ec401c6a95",
+			"sha256": "02c487935c59f495f7e4c26e845b1347c3c24ef390f9e7a56f103ce7da335b1d",
+			"size": 1318758259,
+			"version": "11",
+			"buildtype": "Monthly",
```
Comment (gwolf2u) [inline rubyx.json:14]: cr11 has BETA tag

File: rubyx_changelog.txt [hunk 5]
```diff
@@ -0,0 +1,7 @@
+Highlights & Device Specific Changes:
+Build type: Monthly
```
Comment (gwolf2u) [inline rubyx_changelog.txt:2]: BETA tag

**Review by gwolf2u — CHANGES_REQUESTED** [review]

**Comment by postcumer** [comment]
Closing temporarily

**Pull request closed without merging**